packages/sitecore-jss/src/personalize/graphql-personalize-service.ts

```diff
@@ -68,6 +70,7 @@ export class GraphQLPersonalizeService {
   constructor(protected config: GraphQLPersonalizeServiceConfig) {
     this.config.timeout = config.timeout || 250;
     this.graphQLClient = this.getGraphQLClient();
+    this.memoryCache = new MemoryCacheClient({ cacheTimeout: 10 });
```

Comment:
Should it be configurable? And default value can be `10`

Reply:
Added timeout settings. However, would it make sense to test timeouts in integration tests instead? Since the lowest value for timeout is 1, and we need unit tests to be fast.

Reply:
You can use decimal values to go lower. See example for [dictionary service here](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss/src/i18n/rest-dictionary-service.test.ts#L87).